spec/firebase_app_distribution_api_client_spec.rb

```diff
@@ -144,12 +238,12 @@
     end
 
     it 'does not post when the release notes are empty' do
-      expect(conn).not_to(receive(:post))
+      expect(conn).to_not(receive(:post))
```

Comment:
nit: Remove the parentheses for the `no_not` method so it looks like  `expect(conn).to_not receive(:post)`

Reply:
It looks like rubocop prefers parentheses:

```
spec/firebase_app_distribution_api_client_spec.rb:241:20: C: [Corrected] Use parentheses for method calls with arguments.
      expect(conn).to_not receive(:post)
```